playbooks/inventory/vagrant.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
Oops -- I was testing issues with this script and thought that was causing it. I will put that back.